cmake/HandleGeneralOptions.cmake

```diff
@@ -31,6 +31,13 @@ if(NOT MSVC AND NOT XCODE_VERSION)
     option(GTSAM_BUILD_WITH_CCACHE           "Use ccache compiler cache" ON)
 endif()
 
+# Enable GTSAM_ROT3_EXPMAP if GTSAM_POSE3_EXPMAP is enabled, and vice versa.
```

Comment:
This is a bit problematic as it is done "silently". Add a message?

Reply:
Ideally it should be one flag :slightly_frowning_face: but I need to consult you before making that design change.